Tasks/Common/webdeployment-common/kuduutility.ts

```diff
@@ -190,31 +322,20 @@ async function runCommandOnKudu(publishUrl: string, physicalPath: string, header
     tl.debug('Executing Script on Kudu: ' + kuduDeploymentURL + '. Command: ' + command);
     httpObj.send('POST', kuduDeploymentURL, JSON.stringify(jsonData), headers, async (error, response, body) => {
         if(error) {
-            if(retryFlag && error.toString().indexOf('Request timeout: /api/command') != -1) {
-                console.log(tl.loc('RetryForTimeoutIssue'));
+            if(error.toString().indexOf('Request timeout: /api/command') != -1) {
```

Comment:
I guess utility can have two methods runcommand sync and runcommand async. Waiting for the response and reporting timeout should be sync command. Async should just execute and return. Callers of async should take the responsibility of pooling whatever is appropriate for them.